### ISSUE TITLE
Ask for more information for uploaded files

### ISSUE THREAD

**Issue by srish** [tonythomas01/gdrive_to_commons#8]
There is a bit more information that https://commons.wikimedia.org/wiki/Special:UploadWizard asks for at the time of uploading a file. We have not considered this in the tool as of now. 

It might be useful to allow users to choose, for example, a license, category, date work was created or published, etc.

**Comment by tonythomas01** [comment]
Exactly. We need to see what mwclient supports. From the docs, I see: 
```
            filename (str): Destination filename, don't include namespace
                            prefix like 'File:'
            description (str): Wikitext for the file description page.
            comment (str): Upload comment. Also used as the initial page text
```

**Comment by tonythomas01** [comment]
We have filename and description now. Comments need to be supported.

**Comment by tonythomas01** [comment]
Sounds like we are missing license details, which seems to be a problem.

**Comment by NdibeRaymond** [comment]
Hello, I am an outreachy intern and would love to take up this task @srish
